### PR TITLE
feat(Drawer): init drawer

### PR DESCRIPTION
**Pull request by allblueee**
#7

### CODE REVIEW

**Comment by allblueee** [comment]
#7

**Comment by FliPPeDround** [comment]
Sorry, this PR looks as nothing has been submitted, if you want to claim the component, you can do so at #7

**Comment by FliPPeDround** [comment]
> Sorry, this PR looks as if nothing has been submitted, if you want to claim the component, you can do so at #7

OK，I find it

**Comment by allblueee** [comment]
Sorry, I made a mistake at first

File: src/components/Feedback/Drawer/drawer.story.vue [hunk 1]
```diff
@@ -0,0 +1,421 @@
...
+  )}${isAttribute(
+    basicDrawerData.size !== '30%',
+    `size="${basicDrawerData.size}"`,
+  )}${isAttribute(
+    basicDrawerData.title !== '',
+    `title="${basicDrawerData.title}"`,
+  )}${isAttribute(
+    !basicDrawerData.withHeader,
+    `with-header=${basicDrawerData.withHeader}`,
+  )}>\n<span>Hi, there!</span>\n</el-drawer>`
```
Comment (FliPPeDround) [inline src/components/Feedback/Drawer/drawer.story.vue]: here the indentation is missing

File: src/components/Feedback/Drawer/drawer.story.vue [hunk 2]
```diff
@@ -0,0 +1,421 @@
...
+  directionList,
+  showClose: true,
+  size: '30%',
+  title: 'Drawer',
+  withHeader: true,
+})
+
+const basicSource = computed(() => {
+  return `<el-button>Open the drawer</el-button>
+<el-drawer${isAttribute(
```
Comment (FliPPeDround) [inline src/components/Feedback/Drawer/drawer.story.vue]: `v-model` is missing

File: src/components/Feedback/Drawer/drawer.story.vue [hunk 3]
```diff
@@ -0,0 +1,421 @@
...
+  direction: 'rtl',
+  directionList,
+  showClose: true,
+  size: '30%',
+  title: 'Drawer',
+  withHeader: true,
+})
+
+const basicSource = computed(() => {
+  return `<el-button>Open the drawer</el-button>
```
Comment (FliPPeDround) [inline src/components/Feedback/Drawer/drawer.story.vue]: The click event  `@click` is missing here

File: src/components/Feedback/Drawer/drawer.story.vue (hunk 1 above)
Comment (allblueee) [reply src/components/Feedback/Drawer/drawer.story.vue]: I have fixed the indentation. I'm not sure if  it is suitable.

File: src/components/Feedback/Drawer/drawer.story.vue (hunk 3 above)
Comment (allblueee) [reply src/components/Feedback/Drawer/drawer.story.vue]: I have already added `@click`

File: src/components/Feedback/Drawer/drawer.story.vue (hunk 2 above)
Comment (allblueee) [reply src/components/Feedback/Drawer/drawer.story.vue]: I have added the `v-model` attributes

Comment (FliPPeDround) [reply src/components/Feedback/Drawer/drawer.story.vue]: `v-model` should be a specific variable name

image

Comment (allblueee) [reply src/components/Feedback/Drawer/drawer.story.vue]: OK, I might learn the difference.

**Comment by FliPPeDround** [comment]
Here are some ESlint errors, please fix them first, if you use `vscode`, you can install the eslint plugin to help fix，
You can try `pnpm lint:fix`

**Comment by allblueee** [comment]
> Here are some ESlint errors, please fix them first, if you use `vscode`, you can install the eslint plugin to help fix， You can try `pnpm lint:fix`

I'm sorry for the errors. Thanks for your patience！

**Comment by FliPPeDround** [comment]
> > Here are some ESlint errors, please fix them first, if you use `vscode`, you can install the eslint plugin to help fix， You can try `pnpm lint:fix`
> 
> I'm sorry for the errors. Thanks for your patience！

don't feel sorry, thx your PR ;)

**Pull request merged**